specification/src/chapter2.adoc

```diff
@@ -79,41 +64,54 @@ Depending on use case and ecosystem requirements, a RoT can be:
 * A dedicated trusted subsystem (HW RoT), supporting a FW RoT
 
 Using a HW RoT moves critical functions and assets off a Hart to a dedicated
-trusted subsystem, which can provide stronger protection against physical or
-logical attack than a complex Hart.
+trusted subsystem, which can provide stronger protection against physical and logical attack than a complex Hart.
 
 [width=100%]
 [%header, cols="5,20"]
 |===
 | ID#
 | Requirement
 
+| CAT_NNN
+| A secure system SHOULD implement a HW RoT
+
```

Comment:
Should it state - the penultimate RoT in a secure system should be a HW RoT?

Reply:
There is a not already that a system can have multiple trust chains with their own RoT (secondary RoT), but the HW RoT is the primary RoT on a system. I moved that note earlier in the section and clarified.